context.go

```diff
@@ -960,7 +960,7 @@ func getStringFromScope(scope *parser.Scope, v string) (string, scanner.Position
 }
 
 // Clones a build logic module by calling the factory method for its module type, and then cloning
-// property values.  Any values stored in the module object that are not stored in properties
+// property values.  Any values stored in the module object that are not stored in Properties
```

Comment:
I don't think Properties should be capitalized, that makes me think it is a Go type, but we are using it to refer to a class of structs.